console/frontend/src/main/frontend/src/app/views/status/status.component.ts

```diff
@@ -322,18 +326,14 @@ export class StatusComponent implements OnInit, OnDestroy {
   }
 
   private getFreeDiskSpacePercentage(): void {
-    this.appService
-      .getServerInfo()
-      .pipe(first())
-      .subscribe((serverInfo) => {
-        this.serverInfo = serverInfo;
-        this.freeDiskSpacePercentage =
-          Math.round(
-            (serverInfo.fileSystem.freeSpace /
-              serverInfo.fileSystem.totalSpace) *
-              1000,
-          ) / 10;
-      });
+    this.serverInfoService.serverInfo$.pipe(first()).subscribe((serverInfo) => {
+      this.serverInfo = serverInfo;
+      this.freeDiskSpacePercentage =
+        Math.round(
+          (serverInfo.fileSystem.freeSpace / serverInfo.fileSystem.totalSpace) *
+            1000,
+        ) / 10;
+    });
```

Comment:
Doesn't this only run when serverinfo is refreshed?
So if you navigate from another view to the status view, does it still trigger this?

Reply:
That's fair, I was thinking over time it might be nice to have it check every so often but maybe that's a little to extra for this particular issue

Reply:
I can remove the first. Might be usefull for when people are refreshing to see disk space changes

Reply:
That wouldn't hurt no